app/views/posts/_list.html.erb

```diff
@@ -24,3 +23,9 @@
 <% end %>
 <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
 
+
+
+
+
+
+
```

Comment:
why do you have a lot of empty space here?

Reply:
Deleted those empty lines